eggplant_project/settings/base.py

```diff
@@ -15,6 +15,10 @@
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.messages import constants as messages_constants
 
+COOP_NAME = 'KBHFF'
```

Comment:
sure, it's just we don't have apps that define it, so I couldn't see a point in doing that. Once we decide that we will add an app that does define those settings, add it then it will make sense to do it, don't you think?